ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/validation/CommitterModelConformanceValidator.scala

```diff
@@ -182,7 +181,8 @@ private[transaction] class CommitterModelConformanceValidator(engine: Engine, me
         case DamlStateValue.ValueCase.ARCHIVE =>
           // NOTE(JM): Engine only looks up packages once, compiles and caches,
           // provided that the engine instance is persisted.
```

Comment:
Please update this comment to explain we still parse the archive every time.

We may want to introduce a cache for parsing archives.

Reply:
Discussion about caching to be continued here: https://github.com/digital-asset/daml/pull/12168#discussion_r770496675